flybywiresim/aircraft
# fix(fms): move speed limit to performance data

Fixes #[issue_no]

## Summary of Changes

Move the speed limit data to performance data, so that it is flight-plan-specific. This will also allow implementing speed limit modifications in the A380X.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

- Try entering different speed limits for primary and alternate flight plan, make sure they are obeyed in flight.
- Try ENABLE ALTN to make sure the speed limits of the alternate flight plan become those of the primary flight plan.
- In general, everything should behave the same as before.


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
